docs/features.md

```diff
@@ -1,49 +1,55 @@
 Features
 ========
 
-The list below is a mostly exhuastive list of features that
-the bridge supports.
-
-- Bridging Slack Channels
-    - [x] Public
-    - [x] Private
-    - [x] IM (puppeting)
-- Administration
-    - [x] Provisioning API
-    - [x] Admin Room
-    - [ ] User Admin Rooms
-- [x] Metrics - via prometheus
-- Direct Messaging (puppeting) [1]
-    - [x] Matrix -> Slack
-    - [x] Slack -> Matrix
-- Content bridging (in both directions)
-    - [x] Text (m.text)
-    - [x] Formatted Text (m.text html)
-    - [x] Audio/Video (m.audio/m.video)
-    - [x] Files (m.file)
+The list below is a exhuastive list of features that the bridge supports.
+
+Symbols:
+- 🇵 means that the feature is available to puppeted users.
+- 🧪 means that the feature is experimental.
+
+Notes:
+- [1] - Users who are not puppeted are not able to send reactions as themselves.
+- [2] - Matrix replies appear as threads on Slack. Slack threads appear as replies on Matrix.
+
+## Channel Types
+
+The bridge supports connecting all channel types to Matrix rooms.
+
+- Public
+- Private
+- Group Chats 🇵
+- DMs 🇵
+
+## Content Types
+
+These types are supported in both directions.
+
+- Text / Formatted Text
+- Images / Videos / Audio / Files
+- Reactions [1]
 - Redactions
-    - [x] Matrix -> Slack
-    - [x] Slack -> Matrix
-- Reactions
-    - [x] Matrix -> Slack [2]
-    - [x] Slack -> Matrix [3]
 - Edits
```

Comment:
Not working if the Slack message was an image with text, as recently discovered. Bug #486